code/yamil/python/lab_8.py

```diff
@@ -0,0 +1,98 @@
+import re
+
+# players_csv_outcome = [
+# {"name" : "cristiano ronaldo", "current club" : "manchester united", "european league" : "premier league"},
+# {"name" : "lionel messi", "current club" : "paris st germaine", "european league" : "ligue 1"},
+# {"name" : "radamel falcao", "current club" : "rayo vallecano", "european league" : "la liga"}
+# ]
+
+with open("csv_copy.csv", "r") as f:
+    csv = f.read()
+
+lines = csv.split("\n")
+header_2 = lines[0]
+header = lines[0].split(",")
+
+player_list = []
+player_csv = []
+new_line = "\n"
+choices = ["end", "create", "retrieve", "update", "delete", "options"]

```

Comment:
Great to provide the user their options.
* For the future, it can be useful to tell the user they can use `options`.